snap/snapcraft.yaml

```diff
@@ -5,12 +5,76 @@ license: BSD-3-Clause
 grade: stable
 confinement: strict
 contact: solutions-engineering@lists.canonical.com
-summary: DERP Relay Server for Tailscale
+issues: https://github.com/canonical/derper-snap/issues
```

Comment:
I believe you need to add the source code label as well; see https://github.com/canonical/charmed-openstack-exporter-snap/blob/6096b2fda9b29ad6285a2f7bc0c387c80f59efb8/snap/snapcraft.yaml#L123